vipps-recurring-api.md

```diff
@@ -66,20 +66,32 @@ API version: 3.0.0.
 
 The overall flow is:
 
-1. The merchant creates a draft agreement and proposes it to the customer via Vipps.
-2. The customer approves the agreement in Vipps.
-3. The merchant sends a charge request to Vipps at least two days before due date
-4. If the agreement is active, Vipps authorizes the charge.
-5. The customer can find a full overview in Vipps, including a link to the merchant's website.
+1. The merchant creates a draft agreement and proposes it to the customer via Vipps or MobilePay.
+2. The customer approves the agreement in Vipps or MobilePay.
+3. The customer can find a full overview of the agreement in Vipps or MobilePay, including a link to the merchant's website.
+4. The merchant sends a charge request to Vipps or MobilePay at least two days before due date
+4. If the agreement is active, Vipps or MobilePay authorizes the charge.
+5. Charge will be processed on due date.
+
+This diagram shows a simplified flow:
+
+``` mermaid
+flowchart
+    A(*)
+    B(Agreement status: PENDING)
+    A --> |Merchant creates agreement| B
+    B --> |User approves agreement| C1(Agreement status: ACTIVE)
+    B --> |User ignores agreement| C2(Agreement status: EXPIRED)
+    B --> |User declines agreement| C3(Agreement status: STOPPED)
+    C1 --> |Merchant creates charge| D1(Charge status: PENDING)
+    D1 --> |Charge is authorized by Vipps| D2(Charge status: DUE)
```

Comment:
For the DUE status, should we call it something else the "authorized by Vipps?" maybe "charge is visible to user"

Reply:
Good point, did it in master directly 
https://github.com/vippsas/vipps-recurring-api/commit/adea6a8c93cb0511c5d30366fa10540dc4374a8c